tests/test_channel_properties.py

```diff
@@ -0,0 +1,197 @@
+"""Tests for validating channel properties for different basic data types."""
+import random
+
+import pytest
+
+import nidaqmx
+from nidaqmx.constants import (
+    BridgeConfiguration,
+    CounterFrequencyMethod,
+    Edge,
+    ExcitationSource,
+    FrequencyUnits,
+    PowerIdleOutputBehavior,
+    RTDType,
+    ResistanceConfiguration,
+    TemperatureUnits,
+    TerminalConfiguration,
+    TimeUnits,
+    VoltageUnits,
+)
+from nidaqmx.tests.helpers import generate_random_seed
+
+
+class TestChannelPropertyDataTypes(object):
+    """Contains a collection of pytest tests.
+
+    This validates the property getter,setter and deleter methods for different
+    data types of channel properties.
+    """
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_boolean_property(self, any_x_series_device, seed):
```

Comment:
NI's regression testing conventions: https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions

Rename using given/when/then syntax. This may require splitting the test cases and adding local fixtures to share init code.

Reply:
Renamed the tests with the conventional syntax. Added fixtures to share the init code that is useful for tests